report/src/introduction/agile_approach.tex

```diff
@@ -53,17 +55,15 @@ \subsubsection*{Scrum Artifacts and Tools}
 Daily scrums are meetings where progress towards the sprint goal is discussed.
 Each increment will be inspected during \textbf{sprint review}.
 During such a review, progress is examined, the team and its stakeholders discuss any changes, and then figure out what to do next.
+
 Finally, the team conducts a \textbf{sprint retrospective}, where the team examines how their last sprint went, and how they may improve.
 This is part of the quest for continuous improvement.\cite{sutherlandScrumArtDoing2014}
 
-\subsubsection*{Scrum Process}
-In short, the process for working as a Scrum team involves repeatedly taking backlog items and working on them during a sprint.
+In summary, Scrum involves repeatedly taking backlog items and working on them during a sprint.
```

Comment:
I think it makes sense to remove the section, but I really like the aesthetics of having it. It also, to me, improves the reading process by separating the summary & DoDs into their own section. 
`Artifacts and Tools` becomes very long, and its scope gets a little large.

Thoughts?

Reply:
```suggestion
\subsubsection*{Scrum Process}
In summary, Scrum involves repeatedly taking backlog items and working on them during a sprint.
```